content/consulting/brainstorming.mdx

```diff
@@ -16,7 +16,7 @@ solution:
   project: Brainstorming
 callToAction: 'Talk to us about {{TITLE}}'
 testimonialCategories:
-  - testimonialCategory: content/testimonialCategories/brainstorming.mdx
+  - testimonialCategory: content/testimonialCategories/Brainstorming.mdx
```

Comment:
prefer kebab-case filenames as per rules

Reply:
Considering 'Brainstorming' is a single word and all categories are in uppercase, for consistency, should we modify the casing of all testimonials categories to lower-case? and their respective testimonials?

Reply:
yes, kebab case implies lower case
i'd log it as tech debt and fix it later though as it will muddy this pr

Reply:
Done - here is the issue - https://github.com/SSWConsulting/SSW.Website/issues/2098